qt/settingsdialog.py

```diff
@@ -2197,28 +2200,54 @@ def updateExcludeItems(self):
             item = self.listExclude.topLevelItem(index)
             self._formatExcludeItem(item)
 
+
+    def _format_exclude_item_encfs_invalid(self, item):
+        """Modify visual appearance of an item in the exclude list widget to
+        express that the item is invalid.
+
+        See :py:func:`_formatExcludeItem` for details.
+        """
+        # Icon
+        item.setIcon(0, self.icon.INVALID_EXCLUDE)
+
+        # ToolTip
+        item.setData(
+            0,
+            Qt.ItemDataRole.ToolTipRole,
+            _("Disabled because this pattern is not functional in "
+              "mode 'SSH encrypted'.")
+        )
```

Comment:
```suggestion
            0,
            Qt.ItemDataRole.d,
            _("Disabled because d pattern is not functional in "
              "mode 'd encrypted'.")
        )
```